core/src/mast/serialization/mod.rs

```diff
@@ -71,7 +73,7 @@ const MAGIC: &[u8; 5] = b"MAST\0";
 /// If future modifications are made to this format, the version should be incremented by 1. A
 /// version of `[255, 255, 255]` is reserved for future extensions that require extending the
 /// version field itself, but should be considered invalid for now.
-const VERSION: [u8; 3] = [0, 0, 0];
+const VERSION: [u8; 3] = [0, 0, 1];
```

Comment:
@bobbinth I bumped the `MastForest` version because I added advice map serialization. Do we need to parse previous versions of the serialized `MastForest`? Now, we're only parsing if the versions are the same (see the excerpt below from `read_from`)

```rust
        let version: [u8; 3] = source.read_array()?;
        if version != VERSION {
            return Err(DeserializationError::InvalidValue(format!(
                "Unsupported version. Got '{version:?}', but only '{VERSION:?}' is supported",
            )));
        }
```

Reply:
We don't really have a need to handle multiple versions now (for now, we nuke all the previous data with each new testnet deployment). So - I think it is fine to keep it as it was. Once we finalize MAST/package format (i.e., in a month or two), we can start being more diligent with version tracking.

Reply:
Agreed, let's revert this back to `[0, 0, 0]`